parm/config/gefs/config.fcst

```diff
@@ -0,0 +1,366 @@
+export isatmedmf=1
+tbf=""
+if [[ "${satmedmf}" == ".true." ]]; then tbf="_satmedmf" ; fi
+
+#Convection schemes
+export progsigma=".true."
+tbp=""
+if [[ "${progsigma}" == ".true." ]]; then tbp="_progsigma" ; fi
+
+# Radiation options
+export IAER=1011    ; #spectral band mapping method for aerosol optical properties
+export iovr_lw=3    ; #de-correlation length cloud overlap method (Barker, 2008)
+export iovr_sw=3    ; #de-correlation length cloud overlap method (Barker, 2008)
+export iovr=3       ; #de-correlation length cloud overlap method (Barker, 2008)
+export icliq_sw=2   ; #cloud optical coeffs from AER's newer version v3.9-v4.0 for hu and stamnes
+export isubc_sw=2
+export isubc_lw=2
+
+# RRTMGP radiation scheme
+export do_RRTMGP=.false.
+export doGP_cldoptics_LUT=.false.
+export doGP_lwscat=.false.
+
+# LSM configuration
+# NoahMP only
+export iopt_sfc="3"
+export iopt_trs="2"
+
+# Microphysics configuration
+export dnats=0
+export cal_pre=".true."
+export do_sat_adj=".false."
+export random_clds=".true."
+
+case ${imp_physics} in
+    99) # ZhaoCarr
+        export ncld=1
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_zhaocarr${tbf}${tbp}"
+        export nwat=2
+        ;;
+    6)  # WSM6
+        export ncld=2
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_wsm6${tbf}${tbp}"
+        export nwat=6
+        ;;
+    8)  # Thompson
+        export ncld=2
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_thompson_noaero_tke${tbp}"
+        export nwat=6
+
+        export cal_pre=".false."
+        export random_clds=".false."
+        export effr_in=".true."
+        export ltaerosol=".false."
+        export lradar=".false."
+        export ttendlim="-999"
+        export dt_inner=$((DELTIM/2))
+        export sedi_semi=.true.
+        if [[ "${sedi_semi}" == .true. ]]; then export dt_inner=${DELTIM} ; fi
+        export decfl=10
+
+        export hord_mt_nh_nonmono=5
+        export hord_xx_nh_nonmono=5
+        export vtdm4_nh_nonmono=0.02
+        export nord=2
+        export dddmp=0.1
+        export d4_bg=0.12
+        ;;
+    11) # GFDL
+        export ncld=5
+        export FIELD_TABLE="${HOMEgfs}/parm/parm_fv3diag/field_table_gfdl${tbf}${tbp}"
+        export nwat=6
+        export dnats=1
+        export cal_pre=".false."
+        export do_sat_adj=".true."
+        export random_clds=".false."
+        export lgfdlmprad=".true."
+        export effr_in=".true."
+        export reiflag=2
+
+        export hord_mt_nh_nonmono=5
+        export hord_xx_nh_nonmono=5
+        export vtdm4_nh_nonmono=0.02
+        export nord=2
+        export d4_bg=0.12
+        export dddmp=0.1
+        ;;
+    *) echo "Unknown microphysics option, ABORT!" ;;
+esac
+
+# Stochastic physics
+export DO_SPPT=${DO_SPPT:-"NO"}
+export DO_SKEB=${DO_SKEB:-"NO"}
+export DO_SHUM=${DO_SHUM:-"NO"}
+export DO_LAND_PERT=${DO_LAND_PERT:-"NO"}
+export DO_CA=${DO_CA:-"YES"}
```

Comment:
For mem000, DO_CA should be NO or YES?

Reply:
See above, these are just copied from GFS for now. Future PR can update them (so we can see the changes).

When we get there, set cellular automata to whatever v12 is using unless/until we get guidance from MDAB.